piker/ui/_chart.py

```diff
@@ -1011,9 +1014,17 @@ def maxmin():
     tick_size = chart._lc.symbol.tick_size
     tick_margin = 2 * tick_size
 
+    last = time.time()
     async for quotes in stream:
-        for sym, quote in quotes.items():
 
+        now = time.time()
+        period = now - last
+        if period <= 1/_quote_throttle_rate - 0.001:
```

Comment:
Maybe use `timeit` here

https://stackoverflow.com/a/25823885

Reply:
Toss that in #183 if you want.

As discussed in chat, i don't think the faults of `time.time()` are so relevant at display refresh rates 😂 

Also the top answer on that SO does suggest `time.time()`.